provider/provider_yaml_test.go

```diff
@@ -1120,3 +1124,215 @@ func TestFirestoreDatabaseAutoname(t *testing.T) {
 	pt.SetConfig("gcpProj", proj)
 	pt.Up()
 }
+
+func TestEmptyLabels(t *testing.T) {
```

Comment:
How do we expect empty labels to behave? Can that be part of the test name?
a silly example - `TestEmptyLabelsDefaultToCake` - tells me Pulumi should rewrite all empty labels to say "cake".